libmachine/mcnutils/crc.go

```diff
@@ -229,8 +231,21 @@ func (b *B2dUtils) CopyDiskToMachineDir(diskURL, machineName string) error {
 
 	// By default just copy the existing "cached" iso to the machine's directory...
 	if diskURL == "" {
-		log.Infof("Copying %s to %s...", b.path(), machineIsoPath)
+		log.Infof("\n\nCopying %s to %s...\n\n\n", b.path(), machineIsoPath)
 		return CopyFile(b.path(), machineIsoPath)
+	} else if strings.HasPrefix(diskURL, "file:") {
+		// Local disk image so we can avoid copying by making use of backing image feature of qcow2
+		components := strings.Split(diskURL, "://")
+		if len(components) < 2 {
+			return fmt.Errorf("Invalid URL: %s", diskURL)
+		}
+		diskPath := components[1]
```

Comment:
dunno if this should be using https://golang.org/pkg/net/url/ ?

Reply:
I guess.